src/lib/components/PublishButton/modals/SubmitAndPublishReviewModal.js

```diff
@@ -0,0 +1,163 @@
+// This file is part of InvenioRequests
+// Copyright (C) 2023 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/i18next";
+import { Formik } from "formik";
+import _get from "lodash/get";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Trans } from "react-i18next";
+import { ErrorLabel, RadioField } from "react-invenio-forms";
+import { Button, Checkbox, Form, Icon, Message, Modal } from "semantic-ui-react";
+import * as Yup from "yup";
+
+export class SubmitAndPublishReviewModal extends Component {
+  ConfirmSubmitReviewSchema = Yup.object({
+    acceptAccessToRecord: Yup.string().required(i18next.t("You must accept this.")),
```

Comment:
minor: why a string and not bool? Yup allows a value to be validated against another value (e.g. `yup.bool().oneOf([true], 'You must accept this')`